e2etest/go.work

```diff
@@ -0,0 +1,6 @@
+go 1.19
+
+use (
+	./
```

Comment:
I don't think you need this one?

Reply:
we do, if we remove it we'll get
`main module (go.universe.tf/metallb) does not contain package go.universe.tf/metallb/e2etest`

So we need the current module (e2e) and the metallb one in order to consume the api not from the most recent tag but from the version we have in the repo.